src/Diagnostics/define_diagnostics.jl

```diff
@@ -0,0 +1,156 @@
+"""
+    define_diagnostics!(land_model)
+
+Calls add_diagnostic_viariable! for all variables
+"""
+function define_diagnostics!(land_model)
+
+    # Stored in p
+
+    # Albedo
+    add_diagnostic_variable!(
+        short_name = "alpha",
+        long_name = "Albedo",
+        standard_name = "albedo",
+        units = "",
+        comments = "The fraction of incoming radiation reflected by the land surface.",
+        compute! = (out, Y, p, t) -> compute_albedo!(out, Y, p, t, land_model),
+    )
+
+    # Net radiation
+    add_diagnostic_variable!(
+        short_name = "rn",
+        long_name = "Net Radiation",
+        standard_name = "net_radiation",
+        units = "W m^-2",
+        comments = "Balance between incoming and outgoing shortwave and longwave radiation at the land surface.",
```

Comment:
"Difference" instead of balance?. Maybe we can also note that a positive value means radiation is absorbed by land?

Reply:
I am not sure where, but we should say somewhere that our sign convention throughout is relative to atmosphere or land, e.g., positive if atmosphere gain, negative if atmosphere loss... (is our sign actually consistent for all? e.g., negative values for rain?)